data/loadingUnits.json

```diff
@@ -0,0 +1,59 @@
+{
+  "reference": "Ref XYZ",
+  "load": {
+    "bodyType": "TENT",
+    "equipmentExchange": false,
+    "length": 13.6,
+    "weight": 24
+  },
+  "comment": "An order with some loading units",
+  "loadingUnits": [
+    {
+      "type": "CUSTOM",
+      "icon": "EURO",
+      "name": "EURO",
```

Comment:
So just to clarify, we only rely on `icon` and `name` now to display the loading unit (except for this [bug](https://github.com/impargo/reusable-components/pull/703) in the old PDFs)

So we could safely modify the backend to make `type` optional in the LoadingUnitInput type.

- `name` is used as a display name for the loading unit (sometimes translated if applicable)
- `icon` is used to show one of the supported icons if possible
- `template` is only used as a reference to the template used to add the loading unit. It's only useful when editing loading units to select the correct template (see screenshot)

![image](https://github.com/impargo/nodejs-api-example/assets/15192797/65104a16-f1d3-4a23-b498-ae01d1f1d050)

Reply:
Okay nice thanks for the feedback. I've adapted the example and the backend accordingly.